front/src/modules/command-menu/components/CommandItemText.tsx

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+
+export type CommandItemTextProps = {
+  text: string;
+};
+
+const StyledCommandTextContainer = styled.div`
+  display: flex;
+  gap: ${({ theme }) => theme.spacing(1)};
+  justify-content: center;
+`;
+
+const StyledCommandText = styled.div`
+  color: ${({ theme }) => theme.font.color.light};
+  padding-left: ${({ theme }) => theme.spacing(2)};
+  padding-right: ${({ theme }) => theme.spacing(2)};
+  white-space: nowrap;
+`;
+
+const StyledCommandKeys = styled.div`
+  background-color: ${({ theme }) => theme.background.secondary};
+  border: 1px solid ${({ theme }) => theme.border.color.strong};
+  border-radius: ${({ theme }) => theme.border.radius.sm};
+  box-shadow: 0px 0px 1px ${({ theme }) => theme.boxShadow.strong};
+  padding-left: ${({ theme }) => theme.spacing(1)};
+  padding-right: ${({ theme }) => theme.spacing(1)};
+`;
+
+export const CommandItemText = ({ text }: CommandItemTextProps) => {
```

Comment:
Let's not parse the text but create custom props for our use case.

Example : 

{ 
  firstKey: string
  secondKey: string
}

And let the component handle the "then" text.

Reply:
Should I also pass "then" using props 
There might be case where simultaneous key pressing is required like Ctrl + N
So here it requires '+' instead of 'then'